library/sl

```diff
@@ -15,4 +15,4 @@ Tags: 6
 Architectures: amd64
 GitFetch: refs/heads/6
 Directory: sl6
-GitCommit: 17babe8c23b74cc51ec4bce25d789157650aa8e9
+GitCommit: 042f97be7cece26d56892893bd80a1e48d7b0cd7
```

Comment:
```suggestion
GitCommit: 5288d401c76c12025e0495265067ad6a912b59ce
```

Update to reflect the new commit id in https://github.com/scientificlinux/sl-docker/commits/6